tests/test_models/test_nn/test_chronos.py

```diff
@@ -0,0 +1,277 @@
+    with pytest.warns(UserWarning, match="Actual length of a dataset is less that context size."):
+        _ = pipeline.forecast()
+
+
+@pytest.mark.smoke
+@pytest.mark.parametrize(
+    "model",
+    [
+        ChronosModel(model_name="chronos-t5-tiny", encoder_length=10, limit_prediction_length=False),
+        ChronosBoltModel(model_name="chronos-bolt-tiny", encoder_length=10, limit_prediction_length=False),
+    ],
+)
+def test_forecast_warns_big_prediction_length(ts_increasing_integers, model):
+    if isinstance(model.get_model(), ChronosModelForForecasting):
+        config_prediction_length = model.get_model().config.prediction_length
+    else:
+        config_prediction_length = model.get_model().chronos_config.prediction_length
+    pipeline = Pipeline(model=model, horizon=65)
+    pipeline.fit(ts_increasing_integers)
+    with pytest.warns(UserWarning, match=f"We recommend keeping prediction length <= {config_prediction_length}."):
+        _ = pipeline.forecast()
+
+
+@pytest.mark.smoke
+@pytest.mark.parametrize(
+    "model",
+    [
+        ChronosModel(model_name="chronos-t5-tiny", encoder_length=10, limit_prediction_length=True),
+        ChronosBoltModel(model_name="chronos-bolt-tiny", encoder_length=10, limit_prediction_length=True),
+    ],
+)
+def test_forecast_error_big_prediction_length(ts_increasing_integers, model):
+    if isinstance(model.get_model(), ChronosModelForForecasting):
+        config_prediction_length = model.get_model().config.prediction_length
+    else:
+        config_prediction_length = model.get_model().chronos_config.prediction_length
+    pipeline = Pipeline(model=model, horizon=65)
+    pipeline.fit(ts_increasing_integers)
+    with pytest.raises(ValueError, match=f"We recommend keeping prediction length <= {config_prediction_length}. "):
+        _ = pipeline.forecast()
+
+
+@pytest.mark.parametrize(
+    "model",
+    [
+        ChronosModel(model_name="chronos-t5-tiny", encoder_length=10, num_samples=5),
+        ChronosBoltModel(model_name="chronos-bolt-tiny", encoder_length=10),
+    ],
+)
+def test_forecast(ts_increasing_integers, expected_ts_increasing_integers, model):
+    pipeline = Pipeline(model=model, horizon=1)
+    pipeline.fit(ts_increasing_integers)
+    forecast = pipeline.forecast()
+    assert_frame_equal(forecast.df, expected_ts_increasing_integers.df, atol=2)
+
+
+@pytest.mark.parametrize(
+    "model",
+    [
+        ChronosModel(model_name="chronos-t5-tiny", encoder_length=10, num_samples=1),
+        ChronosBoltModel(model_name="chronos-bolt-tiny", encoder_length=10),
+    ],
+)
+@pytest.mark.parametrize("ts", ["example_tsds", "example_tsds_int_timestamp"])
+def test_forecast_prediction_intervals(ts, model, request):
+    quantiles = [0.1, 0.9]
+    ts = request.getfixturevalue(ts)
+    pipeline = Pipeline(model=model, horizon=1)
+    pipeline.fit(ts=ts)
+    forecast = pipeline.forecast(prediction_interval=True, quantiles=quantiles)
```

Comment:
Shouldn't we check presence of intervals and their relation to `target`?

Reply:
About relation. It is usually expected that:
- upper quantile >= target
- lower quantile <= target

However, it could not be the case for a very skewed distribution. But at least upper quantile >= lower quantile.

We could also check the names of the columns instead of checking only the quantity of them.